python/servo/package_commands.py

```diff
@@ -630,12 +631,33 @@ def upload_to_s3(platform, package, timestamp):
             extension = path.splitext(path.basename(package))[1]
             latest_upload_key = '{}/servo-latest{}'.format(nightly_dir, extension)
 
+            # Compute the hash
+            SHA_BUF_SIZE = 1048576  # read in 1 MiB chunks
+            sha256_digest = hashlib.sha256()
+            with open(package, 'rb') as package_file:
+                while True:
+                    data = package_file.read(SHA_BUF_SIZE)
+                    if not data:
+                        break
+                    sha256_digest.update(data)
+            package_hash = sha256_digest.hexdigest()
+            package_hash_fileobj = io.BytesIO(package_hash)
+            package_hash_upload_key = '{}/{}.sha256'.format(nightly_dir, filename)
```

Comment:
I don't think it's important to preserve hashes for each nightly build. Let's just upload the latest hash.

Reply:
Okay, sure!